libs/s25main/Settings.cpp

```diff
@@ -224,7 +224,7 @@ void Settings::Load()
         video.fullscreenSize.width = iniVideo->getIntValue("fullscreen_width");
         video.fullscreenSize.height = iniVideo->getIntValue("fullscreen_height");
         video.fullscreen = iniVideo->getBoolValue("fullscreen");
-        video.vsync = iniVideo->getBoolValue("vsync");
+        video.framerate = iniVideo->getIntValue("framerate");
```

Comment:
This fails for existing configs using the old name, better provide a default:

```suggestion
        video.framerate = iniVideo->getValue("framerate", 0);
```